tradable-app/src/components/App/App.js

```diff
@@ -0,0 +1,31 @@
+import React, { useContext, useEffect } from "react";
+import Context from "../../store/Context";
+import SideMenu from "../SideNav/SideNav";
+import TopMenu from "../TopMenu/TopMenu";
+import Dashboard from "../../pages/Dashboard/Dashboard";
+import { Container, Main } from "./style";
+import io from "socket.io-client";
+const ENDPOINT = "http://localhost:5000/";
+
+function App() {
+   const { darkMode, getMode, setCandles } = useContext(Context);
+
+   useEffect(() => {
+      const socket = io(ENDPOINT);
+      socket.on("dataReady", (data) => {
```

Comment:
please open a new folder called services and inside a file called sockets so all the socket calls we be there